cetmix_tower_server/tests/test_command.py

```diff
@@ -529,6 +529,16 @@ def test_user_access_rule(self):
         # Ensure that user can access the command
         command_name = test_command_1_as_bob.name
         self.assertEqual(command_name, "Test command", msg="Must return 'Test command'")
+        # Check that user with "cetmix_tower_server.group_user" can execute ssh command
+        test_command.write(
```

Comment:
I think we need to update [this method](https://github.com/cetmix/cetmix-tower/blob/c428b9e07d021f38d09889d7cf898b4c9d005d3c/cetmix_tower_server/tests/common.py#L219) too. And add some read operations to ensure that they don't trigger any access issues.
However let's merge https://github.com/cetmix/cetmix-tower/pull/181 first